src/locales/ua/error.json

```diff
@@ -9,6 +9,7 @@
   "onlyLetter": "Поле повинно містити лише літери",
   "emailLength": "Поле повинно містити від 8 до 60 символів",
   "wrongFormat": "Некоректний формат",
+  "replyCommentField": "Поле повинно містити від 2 до 700 символів",
```

Comment:
I found the same translation inside Bohdan's (Yavdoshniak) pull request. Please pull changes and delete unnecessary fields